app/vmalert/rule/web.go

```diff
@@ -33,7 +33,7 @@ type ApiGroup struct {
 	// Interval is the Group's evaluation interval in float seconds as present in the file.
 	Interval float64 `json:"interval"`
 	// LastEvaluation is the timestamp of the last time the Group was executed
-	LastEvaluation time.Time `json:"lastEvaluation"`
+	LastEvaluation time.Time `json:"lastEvaluation,omitempty"`
```

Comment:
Why having `omitempty` for web API object fields, including existing ones like `json:"annotations,omitempty"`.
It seems not to be used. And it also would be bad if we do ignore fields that are empty in API output, right?

Reply:
exactly for this properly it's needed to simplify parsing logic on frontend side. it's easier to parse empty string rather than `0001-01-01T00:00:00Z`